packages/utils/src/date-utils/getTimeFromPrevious.ts

```diff
@@ -1,45 +1,34 @@
-import { checkIsNextDay } from './checkIsNextDay.js';
+import type { OntimeEvent } from 'ontime-types';
+
 import { dayInMs } from './conversionUtils.js';
 
 /**
  * Utility returns the time elapsed (gap or overlap) from the previous
  * It uses deconstructed parameters to simplify implementation in UI
```

Comment:
suggestion: this statement is no longer true and we would prefer removing the comment